packages/react/src/components/Table/TableHead/_table-head.scss

```diff
@@ -20,6 +20,7 @@
     display: flex;
     user-select: none;
     overflow: hidden;
+    flex-wrap: wrap;
```

Comment:
This class is also used by the span elements inside the normal table columns

![image](https://user-images.githubusercontent.com/5167091/124828542-b2df1a00-df77-11eb-93a2-fe9c56735be6.png)


So we should make sure it is a child of the `iot--column-header-row--table-header` selector and added to the file `_column-header-row.scss` instead, e.g.

```
.#{$iot-prefix}--column-header-row--table-header .#{$prefix}--table-header-label {
  flex-wrap: wrap;
}
```

Reply:
Thanks @bjornalm, for reviewing the PR. I have made the changes as suggested.
https://deploy-preview-2489--carbon-addons-iot-react.netlify.app/?path=/story/1-watson-iot-table-table--with-resize-and-no-initial-columns